85.md

```diff
@@ -0,0 +1,52 @@
+NIP-85
+======
+
+Reviews
+-------
+
+Kind `1986` MAY be used to leave a review with optional ratings.
+
+The `content` field SHOULD include a human-readable review.
+
+The review event MUST include one or more `e`, `a`, `p`, `t`, or `r` tags indicating
+the object of the review. This allows for reviews of events, people, topics, and relays
+or external resources. As with NIP-01, a relay hint SHOULD be included when using `e` and
+`p` tags.
+
+Tags MAY contain additional `rating` tags each of which should have a `value` between 0 and 1,
+and an optional `mark` specifying an attribute of the thing being reviewed.
+
+Tags MAY contain NIP 32 labels to help categorize what is being reviewed.
+
+Examples
+--------
+
+A relay review with rating metadata.
+
+```json
+{
+  "kind": 1986,
+  "content": "This relay is fast!",
+  "tags": [
+    ["L", "review"],
```

Comment:
`t` tags should always be user-provided. Mixing user generated content with semantic data is a recipe for disaster.

I'm sympathetic about `l` tags, because you're probably right, 1986 will probably always have `L=review`. But NIP 32 requires `L` for good reasons, so you can't really omit it. And you do need to identify the type of review in an indexable way, and I can't think of a better tag than `l`. So I don't really see what else you could do here apart from introducing yet another tag.